src/shared/user-menu.jsx

```diff
@@ -128,9 +134,7 @@ export default class UserMenu extends React.Component {
                         Go to Profile
                       </MenuItem>
                       <MenuDivider />
-                      <MenuItem onClick={this.logout}>
-                        Log Out
-                      </MenuItem>
+                      {logout}
```

Comment:
I'd prefer an approach like:

```jsx
{
  !USE_OPENIDC_AUTH && (
    <MenuItem onClick={this.logout}>
       Log Out
    </MenuItem>
  )
}
```

This is more direct, understandable, and compact.